src/model/generate_model.jl

```diff
@@ -185,6 +188,7 @@ function generate_model(setup::Dict,inputs::Dict,OPTIMIZER::MOI.OptimizerWithAtt
 	end
 
 	# Policies
+	co2!(EP, inputs) 
```

Comment:
isn't the CO2 module not a policy?

Reply:
you are right. I moved co2 module to line 129 (after fuel.jl)